examples/step-31/step-31.cc

```diff
@@ -2081,8 +2081,13 @@ namespace Step31
     temperature_solution = tmp[0];
     old_temperature_solution = tmp[1];
 
+    temperature_constraints.distribute(temperature_solution);
```

Comment:
The surrounding comments have been slightly adjusted. I would appreciate any suggestions you might have.